project/frontend/src/App.js

```diff
@@ -37,11 +39,62 @@ function App() {
     setUser((prevState) => {
       return {
         ...prevState,
-        id: 123,
+        id: 1,
       };
     });
   }
 
+  React.useEffect(() => {
+    if (user.signedIn && user.id && compId) {
+      Promise.all([
```

Comment:
what happens if one for them fails?

Reply:
If some fail, then the ones that failed will not be displayed. The successful ones will still be fine.